### ISSUE TITLE
Missing Screen with QT Grabber

### ISSUE THREAD

**Issue by Pacerino** [hyperion-project/hyperion.ng#1403]
- [x] I confirm that this is an issue rather than a question.

#### Steps to reproduce
1. Set screen grabber to QT and choose ALL DISPLAYS

#### What is expected?
Both screen should be visible and recognized by Hyperion.
![qQCyEWVTk6](https://user-images.githubusercontent.com/22194063/147884357-cf913ec8-a615-4283-9925-2af0cd1d5c3d.png)

#### What is actually happening?
I have two screens, one left and one right. Windows recognizes my left screen as my first and the right as my second. However, the second screen is my main screen. So unless I change the screen order in Windows settings, only one screen is recorded and sent to the LEDs.

#### System
```
Hyperion Server: 
- Build:           (HEAD detached at df149583) (GitHub-dc6aa4dd/df149583-1637501177)
- Build time:      Nov 21 2021 16:50:38
- Git Remote:      https://github.com/hyperion-project/hyperion.ng
- Version:         2.0.12
- UI Lang:         de (BrowserLang: de)
- UI Access:       default
- Avail Capt:      v4l2,qt,dx
- Config path:     C:/Users/**REMOVED**/.hyperion
- Database:        read/write

Hyperion Server OS: 
- Distribution:   Windows 10 Version 2009
- Architecture:   x86_64
- Kernel:         winnt (10.0.19043 (WS: 64))
- Root/Admin:     true
- Qt Version:     5.15.0
- Python Version: 3.10.0
- Browser:        Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:95.0) Gecko/20100101 Firefox/95.0
```

**Comment by Pacerino** [comment]
Little Update. I have to change my mainscreen to the one on the left or change the order of the screens. A small Workaround for me now, is to change my Mainscreen to the one on the left. It bothers me a little bit because all my games starting now on the left, but it makes the QT Grabber for both screens working.

**Comment by Pacerino** [comment]
Any updates on this? The workaround is pretty annoying for me...

**Comment by ImAvix** [comment]
+1 on this, this looks like a similar issue to [my issue thread](https://github.com/hyperion-project/hyperion.ng/issues/1320), and that's been open since September last year with no response.

**Comment by Pacerino** [comment]
I have since switched to a fork of Lightpack. I don't know how it works with your monitor setup. You can try it.
https://github.com/psieg/Lightpack

**Comment by Lord-Grey** [comment]
I found some issue and will have a look in providing a fix.
That might address issues in #1320  too.

I will keep you posted....

**Comment by Lord-Grey** [comment]
@Pacerino, @ImAvix I had a go to fix this.
Would you mind testing the fix, please?

A development build can be downloaded from [here](https://github.com/hyperion-project/hyperion.ng/actions/runs/1802718551)
Just look at the Artifact section.

**Comment by Pacerino** [comment]
@Lord-Grey That fixes my problem! Both screens work now!
![grafik](https://user-images.githubusercontent.com/22194063/154779365-cc4e941b-e50b-4eb3-b9fb-18c85c29ab53.png)

**Comment by Lord-Grey** [comment]
Fixed via #1421
